src/SDKs/Azure.Core/data-plane/Azure.Core/Net/Pipeline/HttpPipelineTransport.cs

```diff
@@ -144,28 +143,31 @@ public override void Dispose()
             public override string ToString() =>
                 _responseMessage!=null? _responseMessage.ToString() : _requestMessage.ToString();
 
+            readonly static HttpMethod s_patch = new HttpMethod("PATCH");
             public static HttpMethod ToHttpClientMethod(PipelineMethod method)
             {
                 switch (method) {
                     case PipelineMethod.Get: return HttpMethod.Get;
                     case PipelineMethod.Post: return HttpMethod.Post;
                     case PipelineMethod.Put: return HttpMethod.Put;
                     case PipelineMethod.Delete: return HttpMethod.Delete;
+                    case PipelineMethod.Patch: return s_patch;
```

Comment:
How is patch different than `Post`?

Reply:
https://stackoverflow.com/questions/31089221/what-is-the-difference-between-put-post-and-patch

Reply:
Thank you :)